src/app/map/[id]/Canvas/index.static.tsx

```diff
@@ -11,8 +12,9 @@ export interface StaticMapCanvasProps {
   expandedItemIds?: string[];
   baseHexSize?: number;
   children?: ReactNode;
-  pathname: string;
-  currentSearchParamsString: string;
+  rootItemId: number;
+  userId: number;
+  groupId: number;
```

Comment:
center, rootItemId, userId and groupId are all information about the center. I think you should refactor this to live in a single prop "CenterInfo"